lib/session.js

```diff
@@ -0,0 +1,100 @@
+const { strict: assert } = require('assert');
```

Comment:
Encryption and signing was added and reviewed here:

https://github.com/joshcanhelp/express-openid-connect/pull/1